client/v3/client.go

```diff
@@ -112,7 +113,7 @@ func New(cfg Config) (*Client, error) {
 // service interface implementations and do not need connection management.
 func NewCtxClient(ctx context.Context) *Client {
 	cctx, cancel := context.WithCancel(ctx)
-	return &Client{ctx: cctx, cancel: cancel, lg: zap.NewNop()}
+	return &Client{ctx: cctx, cancel: cancel, lgMu: new(sync.RWMutex), lg: zap.NewNop()}
```

Comment:
@chaochn47 It seems that you missed this change in the second commit in https://github.com/etcd-io/etcd/pull/16800?

Reply:
Thanks for capturing that! 

Zero value of `RWMutex` should be the same `new(sync.RWMutext)` though.

Reply:
> Zero value of RWMutex should be the same new(sync.RWMutext) though.


It isn't true. zero value for a pointer is nil,  while the result of `new(sync.RWMutext)` isn't nil.

Reply:
You are right.